sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -228,4 +233,126 @@ public HttpPipeline getHttpPipeline() {
             });
     }
 
+
+    //==================================================================================================================================================
+    // Models APIs
+    //==================================================================================================================================================
+    
+    /**
+     * Creates one or many models.
+     * @param models The list of models to create. Each string corresponds to exactly one model.
+     * @return A {@link PagedFlux} of created models.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedFlux<ModelData> createModels(List<String> models) {
```

Comment:
For the time being I am using List instead of Iterable as the protocol layer also accepts a List, not sure what the value add is for us to use Iterables

Reply:
I agree, we should stick to `List`.